content/docs/guides/rust.md

```diff
@@ -22,18 +22,26 @@ To create a Neon project:
 
 ## Configure the connection
 
-Add the Neon connection details to your `main.rs` file.
+Add the Neon connection details to your `main.rs` file, as in the following example:
 
 ```rust
-use postgres::{Client, NoTls};
+use postgres::Client;
+use openssl::ssl::{SslConnector, SslMethod};
+use postgres_openssl::MakeTlsConnector;
+use std::error;
 
-fn main() {
- let mut client = Client::connect("user=<user> dbname=<dbname> host=pg.neon.tech password=<password>", NoTls).expect("connection error");
+fn main() -> Result<(), Box<dyn error::Error>> {
+    let builder = SslConnector::builder(SslMethod::tls())?;
+    let connector = MakeTlsConnector::new(builder.build());
 
- for row in client.query("select version()", &[]).expect("query error") {
-     let version: &str = row.get(0);
-     println!("version: {}", version);
- }
+    let mut client = Client::connect("postgres://<user>:<password>@ep-cold-shadow-123456.eu-central-1.aws.neon.tech/<dbname>", connector)?;
```

Comment:
Replace the `ep-cold-shadow-123456.eu-central-1.aws.neon.tech` with `<endpoint_hostname>` how it's done for Go client

Reply:
Good point. Thank you. I have submitted and update.